### PR TITLE
Allow reverting (changing) between assets

### PR DESCRIPTION
**Pull request by ShaunYearStrong**
picture 29

### CODE REVIEW

File: public/video-ui/src/services/VideosApi.js [hunk 1]
```diff
@@ -40,6 +40,13 @@ export default {
     })
   },
 
+  revertAsset: (version, videoId) => {
+    return pandaReqwest({
+      url: '/api/atom/' + videoId + '/revert/' + version,
+      method: 'post'
```
Comment (akash1810) [inline public/video-ui/src/services/VideosApi.js]: a `PUT` would suit this endpoint more imo, but not related to this PR

File: public/video-ui/src/components/VideoAssets/VideoAssetItem.js [hunk 2]
```diff
@@ -9,12 +9,29 @@ export default class VideoAssetItem extends React.Component {
...
+
+  renderAssetVersionButton = () => {
+    if(this.props.activeAsset !== this.props.asset.version) {
+      return (
+        <button type="button" className="btn" onClick={this.revertAsset}>
+          Set as current asset
+        </button>
+      )
+    } else {
+      return false;
```
Comment (akash1810) [inline public/video-ui/src/components/VideoAssets/VideoAssetItem.js]: don't know much react - would `false` get rendered below?

File: public/video-ui/src/services/VideosApi.js (hunk 1 above)
Comment (ShaunYearStrong) [inline public/video-ui/src/services/VideosApi.js]: Yep, agreed!

File: public/video-ui/src/components/VideoAssets/VideoAssetItem.js (hunk 2 above)
Comment (ShaunYearStrong) [inline public/video-ui/src/components/VideoAssets/VideoAssetItem.js]: Nah, if the render function doesn't return anything, then nothing happens

**Pull request merged**